website/docs/docs/build/sql-models.md

```diff
@@ -262,7 +262,7 @@ Additionally, the `ref` function encourages you to write modular transformations
 
 ## Testing and documenting models
 
-You can also document and test models &mdash; skip ahead to the section on [testing](/docs/build/tests) and [documentation](/docs/collaborate/documentation) for more information.
+You can also document and test models &mdash; skip ahead to the section on [testing](/docs/build/data-tests) and [documentation](/docs/collaborate/documentation) for more information.
```

Comment:
We may want to have an overall "test" page that includes a quick overview of both data tests and unit tests to link to in cases like this

Reply:
Yes! I will create that when we're documenting the unit testing narrative